helm-charts/basehub/values.yaml

```diff
@@ -30,6 +30,9 @@ inClusterNFS:
   enabled: false
   size: 100Gi
 
+docs_service:
+  enabled: false
+
```

Comment:
This is the default value for use here:

https://github.com/2i2c-org/infrastructure/blob/5d316ac8ad1cc2faf3b5a1323214f353d5c911e4/helm-charts/basehub/templates/docs-service-deployment.yaml#L1-L4

We could also make it so, that we rely directly on `jupyterhub.custom.docs_service.enabled`, making us not duplicate code. If we do need to configure this in two locations, we should have inline comments about the need to update one if the other.

I note that there are other configuration values of relevance to docs_service, such as...

https://github.com/2i2c-org/infrastructure/blob/5d316ac8ad1cc2faf3b5a1323214f353d5c911e4/helm-charts/basehub/templates/docs-service-deployment.yaml#L26-L34

So, they need a home as well, maybe put them under `jupyterhub.custom.docs_service` as well? I'm not sure what I think makes most sense.

Could you as part of this PR add the default values for any configuration related to the docs_service (`enabled`, `branch`, `repo` --- I figure they are `false`, `""`, and `""`)

Reply:
> So, they need a home as well, maybe put them under jupyterhub.custom.docs_service as well? I'm not sure what I think makes most sense.

I asked the same thing in a comment above :sweat_smile: Let's go for that since we both thought about it.

> Could you as part of this PR add the default values for any configuration related to the docs_service (enabled, branch, repo --- I figure they are false, "", and "")

Sure!